guidance/be-nice.md

```diff
@@ -0,0 +1,53 @@
+# Contributors Code of Conduct
+
+The .NET Foundation was created to foster an open, innovative and inclusive
+community around open source .NET. To clarify expected behaviour in our
+communities we have adopted the [Contributor Covenant](http://www.dotnetfoundation.org/code-of-conduct).
+This code of conduct has been adopted by [many other open source communities](http://contributor-covenant.org/adopters/)
+and we feel it expresses our values well. 
+
+For a full copy of the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct),
+see the .NET Foundation Website http://www.dotnetfoundation.org/code-of-conduct
+
+## Applying the Code of Conduct to your project
+
+First of all, it is important that you read the [code of conduct](http://www.dotnetfoundation.org/code-of-conduct)
+and understand when to be on the look out for unnaceptable behaviour, what to do if you see
+it or if someone reports their concerns to you.
+
+Any concerns passed to you by your community should be forwarded to the
+.NET Foundation by email conduct@dotnetfoundation.org for help and advice.
+
+The .NET Foundation recommend that you also display a link to the code of conduct
+in your README.md file and in your CONTRIBUTING.md file if you have one.  You can
+see an example of this in the [Open Live Writer](http://github.com/openlivewriter/openlivewriter)
+project
+
+See the [README.md guidance](readme-guide.md) for the suggested MarkDown to use
+when linking to the Code of Conduct
+
+If introducing the Code of Conduct to an existing project, it is important 
+to do this via a Pull Request to allow the community time to read the code
+of conduct and make sure that they understand it.  Below are some FAQ's you 
+might find useful.
+
+## Code of Conduct F.A.Q's
+
+#### What happens when I report a concern?
+Instances of abusive, harassing, or otherwise unacceptable behavior may be 
+reported by contacting conduct@dotnetfoundation.org. 
+
+All complaints will be reviewed and investigated by two person, gender balanced
```

Comment:
What about race, religion, sexual orientation, etc.? Not sure why we're singling out gender here. Does the composition of that team change depending on what the issue is?